roles/bind-metrics-apb/templates/bind_secret.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "{{ prometheus_secret_name }}-{{ CLIENT_ID }}"
+  namespace: "{{ namespace }}"
+  labels:
+    mobile: enabled
+    name: "{{ prometheus_secret_name }}-{{ CLIENT_ID }}"
+    clientId: "{{ CLIENT_ID }}"
+    serviceName: "{{ prometheus_secret_name }}"
+    serviceInstanceId: "{{ _apb_service_instance_id }}"
+stringData:
+  clientId: "{{ CLIENT_ID }}"
+  config: "{}"
```

Comment:
Is this field used?

Reply:
This is empty for now because there is no content required yet for it but is needed for the client config.

Reply:
didn't we update the UI to default to {} if this field was not present, so that this would not be necessary?

Reply:
I think @sedroche will be updating this with the UI change for [2413](https://issues.jboss.org/browse/AEROGEAR-2413). Need this field for now for verification

Reply:
This needs to go into the `mobileClientConfig` component. As that is part of an upstream PR can you add this @philbrookes ?